pytorch_pretrained_bert/tokenization.py

```diff
@@ -114,6 +126,16 @@ def tokenize(self, text):
           split_tokens = self.wordpiece_tokenizer.tokenize(text)
         return split_tokens
 
+    def encode(self, text):
+        return self.tokenize(text)
+
+    def decode(self, tokens):
+        # Confusing language here: the input "tokens" in the superclass decode()
```

Comment:
Do you want to standardize the language, maybe?
Then it's probably good to do it for all the tokenization files.

Reply:
I can and would like to, but this is sort of the issue with backwards compatibility. In this example, `BertTokenizer` has a method called `tokenize()` that doesn't return IDs, but word tokens. Other things rely upon it.